includes/compat/mailchimp-for-woocommerce.php

```diff
@@ -0,0 +1,26 @@
+<?php
+/**
+ * Newspack setup
+ *
+ * @package Newspack
+ */
+
+namespace Newspack;
```

Comment:
`namespace` declaration seems to make the function unavailable as `"newspack_mc4c_remove_cookie"` for the `add_action` hook:

```
PHP Warning:  call_user_func_array() expects parameter 1 to be a valid callback, function 'newspack_mc4c_remove_cookie' not found or invalid function name […]
```

Removing this line makes the feature work as described.

Reply:
Ah okay, I think I might not have tested that properly after I moved it from my initial test plugin, sorry. :facepalm:

Reply:
Note: it's still an issue, so I'm adding the changes needed label